benchmarks/continous/db/tool/orm/migrations/2024-06-12-120242_refine_ft_transfers/down.sql

```diff
@@ -0,0 +1,23 @@
+-- Assumes regularly values handled here cannot be negative, hence -1 flags out
+-- of range values.
+create function convert_int_to_bigint(x bigint)
```

Comment:
Isn't this the other way around, bigint to int?

Reply:
Fixed in https://github.com/near/nearcore/pull/11563/commits/f6b41bc8bfef26e13b5ad5136314e2723ada20ee.